examples/decoding/plot_receptive_field.py

```diff
@@ -0,0 +1,135 @@
+speech = data['envelope'].T
+sfreq = float(data['Fs'])
+sfreq /= n_decim
+speech = mne.filter.resample(speech, down=n_decim, npad='auto')
+raw = mne.filter.resample(raw, down=n_decim, npad='auto')
+
+
+# Read in channel positions and create our MNE objects from the raw data
+montage = mne.channels.read_montage('biosemi128')
+montage.selection = montage.selection[:128]
+info = mne.create_info(montage.ch_names[:128], sfreq, 'eeg', montage=montage)
+raw = mne.io.RawArray(raw, info)
+n_channels = len(raw.ch_names)
+
+###############################################################################
+# Plot a sample of brain and stimulus activity
+fig, ax = plt.subplots()
+ax.plot(scale(raw[:][0][..., :800].T), color='k', alpha=.1)
+ax.plot(scale(speech[0, :800]), color='r', lw=2)
+ax.set(title="Sample brain and stimulus activity", xlabel="Time (s)")
+mne.viz.tight_layout()
+
+###############################################################################
+# Create and fit a receptive field model
+
+# Define the delays that we will use in the receptive field
+tmin, tmax = -.4, .2
+
+# Initialize the model
+spec_names = ['freq_%s' % ii for ii in range(speech.shape[0])]
+rf = ReceptiveField(tmin, tmax, sfreq,
+                    feature_names=spec_names, model=Ridge(alpha=1.))
+n_delays = int((tmax - tmin) * sfreq) + 2  # +2 to account for 0 + end
+
+n_splits = 3
+cv = KFold(n_splits)
+
+# Prepare model data
+Y, _ = raw[:]  # Outputs for the model
+Y = np.rollaxis(Y, -1, 0)  # Make time the first dimension
+speech = np.rollaxis(speech, -1, 0)
+
+# Iterate through folds, fit the model, and predict/test on held-out data
+coefs = np.zeros([n_splits, n_channels, n_delays])
+scores = np.zeros([n_splits, n_channels])
+for ii, (train, test) in enumerate(cv.split(speech)):
+    print('CV iteration %s' % ii)
+    rf.fit(speech[train], Y[train])
+    preds = rf.predict(speech[test])
+
+    # mask_pred tells us which points were removed / kept in time delaying
+    for jj, i_ch in enumerate(Y.T):
+        scores[ii, jj] = pearsonr(Y[:, jj][test][rf.mask_predict_],
+                                  preds[:, jj])[0]
+    # Remove features dimension because there's only one item in it
+    coefs[ii] = rf.coef_
+delays_time = rf.delays_ / float(sfreq)
+# Average scores and coefficients across CV splits
+mn_coefs = coefs.mean(0)
+mn_scores = scores.mean(0)
+
+# Plot mean prediction scores across all channels
+fig, ax = plt.subplots()
+ix_chs = np.arange(n_channels)
+ax.plot(ix_chs, mn_scores)
+ax.axhline(0, ls='--', color='r')
+ax.set(title="Mean prediction score", xlabel="Channel", ylabel="Score ($r$)")
+mne.viz.tight_layout()
+
+###############################################################################
+# Investigate model coefficients
+#
+# Here we will look at how linear the linear coefficients, sometimes referred
+# to as beta values, are distributed across time delays as well as across the
+# scalp. We will recreate figures 1 and 2C in [1]_
```

Comment:
I can just do that with markdown, ya? e.g. `...will recreate [figure 1](http://journal.frontiersin.org/article/10.3389/fnhum.2016.00604/full#F1)...`

Should I add a `#noqa` in order to account for the fact the URLs will take up too much space?